MyStore/src/MyStore.cpp

```diff
@@ -3,35 +3,61 @@
 #include <chrono>
 #include <windows.h>
 #include <filesystem>
+#include <thread>
+#include <mutex>
 
 using namespace std::chrono;
 
 // This variable is needed to avoid confusion with relative paths in different modules
 // Example of usage:
 // std::string filePath = gProjectPath + "\\data\\Data.json"
+
+void GenerateProducts(StorageData& stor, const int& period, bool& flag);
+void ExitCondition(bool& flag);

```

Comment:
It's better not to split declaration and implementation of local functions in .cpp files. In your case if function signature will change, you'll have to change it in two places instead of one. So let's move implementation here.

Reply:
Make sense!